AGENTS.md

```
@@ -0,0 +1,252 @@
+make clean        # remove .o files and out/
+make editor       # compile → out/super-mango-editor
+make run-editor   # compile + run the level editor
+```
+
+The Makefile uses explicit per-directory wildcards — **new `.c` files in existing source directories are picked up automatically**. New source directories need Makefile entries.
+
+---
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)